### ISSUE TITLE
Implementing array parameters as input 

### ISSUE THREAD

**Issue by vaibhavchellani** [HTTP-APIs/hydrus#200]
### I'm submitting a 
- [ ] bug report.
- [ X] feature request.

### Current Behaviour:

currently using hydrus we cannot accept arrays as inputs in requests .
### Expected Behaviour:

we would be able to handle requests like 'endpoint/[variable 1 , variable 2 , variable 3]

### Do you want to work on this issue?

yes , definitely

**Comment by vaibhavchellani** [comment]
@xadahiya what would be the use case for this feature ?

**Comment by xadahiya** [comment]
We discussed the use cases in detail on gitter. Check that

**Comment by vaibhavchellani** [comment]
@xadahiya in (this)[https://github.com/HTTP-APIs/hydrus/blob/master/hydrus/app.py#L218] if we change the data type of id to array and just iteratively perform the task of updation would it be a good solution ?

**Comment by xadahiya** [comment]
> we would be able to handle requests like 'endpoint/[variable 1 , variable 2 , variable 3]

Don't you think we need to edit get() not post() for this?

**Comment by vaibhavchellani** [comment]
I cannot think of a use case for get() , i have use cases for post where user would want to create or edit multiple objects . To support something like this we need to edit post and put 
```
/user/createWithList:
    post:
      tags:
        - user
      summary:                    Creates list of users with given input array
      description:                ''
      operationId:                createUsersWithListInput
      produces:
        - application/xml
        - application/json
      parameters:
        - in:                     body
          name:                   body
          description:            List of user object
          required:               true
          schema:
            type:                 array
            items:
              $ref:               '#/definitions/User'
      responses:
        default:
          description:            successful operation
```

**Comment by xadahiya** [comment]
> Implementing arrays as inputs for `get()` would mean querying right?

Yeah, as far as I remember we had some cases like `findByStatus` and `findByTags` but I think this will require complex CRUD functionality. So, if you want, you can postpone the get part for now. 

> just iteratively perform the task of updating would it be a good solution?

This can work but it won't be efficient. Sqlalchemy provides functions for bulk operations like `bulk_save_objects()` to do such operations more efficiently. See this http://docs.sqlalchemy.org/en/rel_1_0/orm/persistence_techniques.html#bulk-operations

So, it would be better to write new CRUD functions to perform bulk operations and don't forget to write Tests.

**Comment by vaibhavchellani** [comment]
yes i agree , thank you !

**Comment by xadahiya** [comment]
Fixed with #224
